Signal/src/call/CallService.swift

```diff
@@ -857,6 +857,8 @@ protocol CallServiceObserver: class {
 
         Logger.info("\(TAG) in \(#function): \(call.identifiersForLogs).")
 
+        // TODO make call record.
```

Comment:
I don't think this is controversial, an upcoming PR will insert a "declined call" record.